src/i18n/MControlCenter_tr.ts

```diff
@@ -0,0 +1,230 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.0" language="tr_TR" sourcelanguage="en_US">
+<context>
+    <name>MainWindow</name>
+    <message>
+        <source>Info</source>
+        <translation>Bilgi</translation>
+    </message>
+    <message>
+        <source>EC Version:</source>
+        <translatorcomment>don&apos;t know how to translate this</translatorcomment>
+        <translation>EC Version:</translation>
+    </message>
+    <message>
+        <source>EC Build:</source>
+        <translatorcomment>don&apos;t know how to translate this</translatorcomment>
+        <translation>EC Build:</translation>
```

Comment:
```suggestion
        <translation type="unfinished">EC Build:</translation>
```

In this case, it's better to leave unfinished